### PR TITLE
Add illustrations and examples

### PR DESCRIPTION
**Pull request by alexshalamov**
This PR adds illustrations, examples and fixes constructor parameter for GravitySensor.

***
<a href="https://pr-preview.s3.amazonaws.com/alexshalamov/accelerometer/pull/33.html" title="Last updated on Feb 5, 2018, 11:36 AM GMT (19529fe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/accelerometer/33/9fa57e9...alexshalamov:19529fe.html" title="Last updated on Feb 5, 2018, 11:36 AM GMT (19529fe)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -65,6 +65,51 @@ Examples {#examples}
...
+    console when the [=dom screen=] is perpendicular to the ground and bottom of the
+    rendered web page is pointing downwards.
+    <pre highlight="js">
+    let sensor = new GravitySensor({frequency: 5, referenceFrame: "screen"});
+
+    sensor.onreading = () => {
+      if (sensor.y >= 9.8) {
+        console.log("Web page is perpendicular to the ground.");
+      } else {
+        console.log("Rotate device.");
```
Comment (pozdnyakov) [inline index.bs]: maybe drop this log?

File: index.bs [hunk 2]
```diff
@@ -65,6 +65,51 @@ Examples {#examples}
...
+    let lastAccel = 0;
+    let shakeCount = 0;
+
+    let sensor = new LinearAccelerationSensor({frequency: 60});
+
+    sensor.addEventListener('reading', () => {
+      accel = accel * 0.9 + sensor.x - lastAccel;
+      lastAccel = sensor.x;
+
+      if (accel > shakeThreshold) {
```
Comment (pozdnyakov) [inline index.bs]: isn't  it simpler just `sensor.x > shakeThreashold`  (similar to https://github.com/intel/generic-sensor-demos/blob/master/punchmeter/main.js#L137) ?

File: index.bs (hunk 1 above)
Comment (alexshalamov) [reply index.bs]: sure

File: index.bs (hunk 2 above)
Comment (alexshalamov) [reply index.bs]: Ah, this one has low pass filter and avoids 'recoil' 😄, but for simplicity, sure.

File: index.bs (hunk 1 above)
Comment (alexshalamov) [reply index.bs]: done

File: index.bs (hunk 2 above)
Comment (alexshalamov) [reply index.bs]: done

File: index.bs [hunk 3]
```diff
@@ -65,6 +65,42 @@ Examples {#examples}
...
+    The following example detects shake gesture along x axis of the device, regardless
+    of the orientation of the [=dom screen=].
+    <pre highlight="js">
+    const shakeThreshold = 25;
+
+    let sensor = new LinearAccelerationSensor({frequency: 60});
+
+    sensor.addEventListener('reading', () => {
+      if (sensor.x > shakeThreshold) {
+        console.log(\`Shake detected.\`);
```
Comment (pozdnyakov) [inline index.bs]: why not just "Shake detected." ? :)

Comment (alexshalamov) [reply index.bs]: formatter leftover

Comment (alexshalamov) [reply index.bs]: done

File: index.bs [hunk 4]
```diff
@@ -240,7 +281,7 @@ The GravitySensor Interface {#gravitysensor-interface}
 --------------------------------
 
 <pre class="idl">
-  [Constructor(optional SensorOptions options), SecureContext, Exposed=Window]
+  [Constructor(optional SpatialSensorOptions options), SecureContext, Exposed=Window]
```
Comment (pozdnyakov) [inline index.bs]: Please update also the abstract operation for the GravitySensor construction:
[=construct spatial sensor object=]

Comment (alexshalamov) [reply index.bs]: done

**Comment by alexshalamov** [comment]
@anssiko Could you please take a look? Thanks.

**Comment by alexshalamov** [comment]
thanks!

**Pull request merged**